docs/topics/db/aggregation.txt

```diff
@@ -425,6 +425,27 @@ publisher's that have at least one book with a rating exceeding 3.0. The second
 query asks for the average of a publisher's book's ratings for only those
 ratings exceeding 3.0.
 
+Using multivalued filtering against a previously annotated aggregation might
+still yield duplicate results.
+
+Given:
+
+* Publisher A has two books with each having 100 pages.
```

Comment:
If there is a single "pre-condition", I would suggest to use a single line without the bulleting:
```suggestion
Consider a Publisher named ``A`` that has two books, each with 100 pages.
```

Though the example demonstrating the issue never filters by `A`, so it seems that the `A` is irrelevant for the example? If yes, we should remove that to avoid distracting the reader.